apps/wing/src/commands/pack.ts

```diff
@@ -176,7 +159,10 @@ async function withTempDir<T>(work: (workdir: string) => Promise<T>): Promise<T>
     return await work(workdir);
   } finally {
     process.chdir(cwd);
-    await fs.rm(workdir, { recursive: true });
+    // if you want to debug this you can keep the workdir around
+    if (!process.env.WING_PACK_KEEP_WORKDIR) {
```

Comment:
I think this should be the same as `--no-clean`

Reply:
got it, yeah I wasn't sure if it's something we wanted to add to our public API or not. Wanna propose a set of conventions for debug options / CLI commands in an issue?